rise-dc-app/src/App.tsx

```
@@ -37,6 +38,7 @@ function App() {
           element={<RecipeComplete />}
         />
         <Route path="/scheduler" element={<Scheduler />} />
+        <Route path="/test-event-card" element={<EventCardTest />} />//testing remove after
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)